.github/workflows/ci.yml

```diff
@@ -13,6 +13,11 @@ env:
   CI_ENVIRONMENT: normal
 
 jobs:
+  Get-CI-Image-Tag:
+    uses: opensearch-project/opensearch-build/.github/workflows/get-ci-image-tag.yml@main
```

Comment:
Lets use a published task not a direct repo reference

Reply:
Hm... may you share the example? (PS: this step is replicated to all repos)

Reply:
I don't like tightly coupling these repositories together; security depends on opensearch-build, opensearch build depends on security - this is a formula to get into a broken state and have to start bypassing CI.

Reply:
For example, here is a GitHub action that allows for multi-platform file download [1] it shouldn't be part of the security project even though it was originally written to support this project. 

I have trouble adopting this level of coupling when we still have problems doing normal builds after version bumps [2]

- [1] https://github.com/peternied/download-file
- [2] https://github.com/opensearch-project/security/pull/3829

Reply:
I am still unsure what you are suggesting, my apologies. If this is not the way we should share the workflows, no objections to change it, but to what model?

Reply:
We should publish the workflows as github actions e.g. `opensearch-project/opensearch-ci-image-by-tag@v1`?

Reply:
Oh I see, thanks @peternied , @peterzhuamazon fyi ^^